src/content/posts/4-way-handshake.md/index.md

```diff
@@ -0,0 +1,139 @@
+---
+title: "4-Way Handshake - Wifi Authentication"
+summary: "In this article, you will learn how authentitcation works for WPA/WPA2 network"
+date: 2024-10-10T20:00:00+0200
+lastUpdate: 2024-10-10T20:00:00+0200
+tags: ["network", "wifi"]
+author: clementS
+draft: false
+---
+
+## Abstract
+To connect to a WiFi network, the authentication depends on the WiFi protocol. The most common protocols currently in use are 
+1. **WEP**
+2. **WPA Personal**
+3. **WPA Entreprise**
+4. **WPA2-Personal**
+5. **WPA2-Enterprise**
+6. **WPA3**  
+
+> There are 2 different versions of WPA and WPA2 which work slightly differently in terms of authentication.
+
+While WEP and WPA3 have their own authentication protocols, WPA and WPA2 (which are the most common ones) use the **4-Way Handshake** method to allow a device to connect to WiFi and secure connection.
+
+The 4-Way Handshake requires a PMK (Pair Master Key), a secret shared by both sides (client and server).
+This can be a password (network key) or a username + password as it's the case for Enterprise authentication.
+
+In this article, you will learn how this method works from a network-oriented approach.
+
+## Configuration
+We will consider a WPA-Personal/WPA2-Personal WiFi, so the PMK will just be the WiFi password encrypted. We will go into further explains soon.  
+
+But remember that except getting PMK before authentication starts threw a WiFi password(secret shared by both sides), WPA2-Entreprise and WPA-Entreprise works the same way.
+Indeed, they use 4-way handshake to secure communication.  

```

Comment:
Remember that except for the computing of PMK through/using a WiFi password (secret shared by both sides) WPA2-Entreprise and WPA-Entreprise works the same way. <-- that part is not clear. What is the 'only' difference ?
The both use 4-way handshake to secure communication

Reply:
added infos